Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -163,8 +163,8 @@ open class NavigationMapView: UIView {
     var routeLineGranularDistances: RouteLineGranularDistances?
     var routeRemainingDistancesIndex: Int?
     var fractionTraveled: Double = 0.0
+    var preLocation: CLLocationCoordinate2D?
```

Comment:
I would suggest a bit more descriptive name. It is hard to tell right away what is `pre` prefix here and what this variable is for